src/main/java/ro/hiringsystem/model/PhoneNumberInformation.java

```diff
@@ -0,0 +1,31 @@
+package ro.hiringsystem.model;
+
+import lombok.*;
+
+import java.util.UUID;
+import java.util.function.Predicate;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class PhoneNumberInformation {
```

Comment:
This class is as redundant as MailInformation for the same reasons mentioned there.